src/fparser/two/tests/test_bases.py

```diff
@@ -114,3 +114,21 @@ def test_io_ctrl_spec_errors():
     # description
     obj = Io_Control_Spec.match("not_unit=23")
     assert obj is None
+
+
+def test_blockbase_tofortran_non_ascii(f2003_create):
```

Comment:
It's unclear what part of blockbase this test exercises. Could you please make it more clear, preferably explicitly calling the blockbase method.

Reply:
In fact, it was just repeating the test from Case_Construct so I've removed it altogether.